src/components/Modal/LoginModal.js

```diff
@@ -34,7 +29,7 @@ function LoginModal() {
   };
 
   return (
-    <section>
+    <Dimmer onClick={() => props.clickModal()}>
```

Comment:
```suggestion
    <Dimmer onClick={props.clickModal}>
```
굳이 인라인으로 새로운 함수 선언할 필요 없어보입니다.